honeybee_radiance_command/dctimestep.py

```diff
@@ -0,0 +1,323 @@
+
+    @property
+    def options(self):
+        """dctimestep options."""
+        return self._options
+
+    @options.setter
+    def options(self, value):
+        if not value:
+            value = DctimestepOptions()
+
+        if not isinstance(value, DctimestepOptions):
+            raise ValueError('Expected Dctimestep options not {}'.format(value))
+
+        self._options = value
+
+    @property
+    def sky_vector(self):
+        """Sky vector file that is meant for daylight coefficient and three phase
+        simulations. This is typically generated by gendaymtx or genskyvec"""
+        return self._sky_vector
+
+    @sky_vector.setter
+    def sky_vector(self, value):
+        self._sky_vector = typing.path_checker(value)
+
+    @property
+    def sun_vector(self):
+        """Sun vector file that is meant for direct sun calculations. This is
+        typically generated by gendaymtx or genskyvec"""
+        return self._sun_vector
+
+    @sun_vector.setter
+    def sun_vector(self, value):
+        self._sun_vector = typing.path_checker(value)
+
+    @property
+    def day_coef_matrix(self):
+        """Daylight coefficient matrix file"""
+        return self._day_coef_matrix
+
+    @day_coef_matrix.setter
+    def day_coef_matrix(self, value):
+        self._day_coef_matrix = typing.path_checker(value)
+
+    @property
+    def sun_coef_matrix(self):
+        """Sun matrix file that is meant for direct sun calculations"""
+        return self._sun_coef_matrix
+
+    @sun_coef_matrix.setter
+    def sun_coef_matrix(self, value):
+        self._sun_coef_matrix = typing.path_checker(value)
+
+    @property
+    def view_matrix(self):
+        """View matrix file used for three phase and four phase simulations. Generated
+        by rfluxmtx."""
+        return self._view_matrix
+
+    @view_matrix.setter
+    def view_matrix(self, value):
+        self._view_matrix = typing.path_checker(value)
+
+    @property
+    def t_matrix(self):
+        """Transmission matrix file that is a data-driven BSDF. Typical format is xml"""
+        return self._t_matrix
+
+    @t_matrix.setter
+    def t_matrix(self, value):
+        self._t_matrix = typing.path_checker(value, extn_list=('.xml',),
+                                             file_descr='BSDF file')
+
+    @property
+    def daylight_matrix(self):
+        """Daylight matrix file that is meant for three or four phase simulations."""
+        return self._daylight_matrix
+
+    @daylight_matrix.setter
+    def daylight_matrix(self, value):
+        self._daylight_matrix = typing.path_checker(value)
+
+    @property
+    def facade_matrix(self):
+        """Facade matrix file that is meant for four phase simulations."""
+        return self._facade_matrix
+
+    @facade_matrix.setter
+    def facade_matrix(self, value):
+        self._facade_matrix = typing.path_checker(value)
+
+    @classmethod
+    def for_daylight_coef_calc(cls, options=None, output=None,
```

Comment:
I don't know if we need the `for_` in front of all these `classmethod` names.

We usually put the `classmethods` all the way up after `__init__` but there is no hard rule for it.

Reply:
Okay, I will remove the for: and shift the location of the classmethods.